src/Microsoft.Health.Dicom.Core/Extensions/DicomTagExtensions.cs

```diff
@@ -24,5 +25,28 @@ public static string GetPath(this DicomTag dicomTag)
             EnsureArg.IsNotNull(dicomTag, nameof(dicomTag));
             return dicomTag.Group.ToString("X4") + dicomTag.Element.ToString("X4");
         }
+
+        /// <summary>
+        /// Get default VR for dicom tag.
+        /// </summary>
+        /// <param name="dicomTag">The dicm tag</param>
+        /// <returns>The default VR.</returns>
+        public static DicomVR GetDefaultVR(this DicomTag dicomTag)
```

Comment:
Should'nt all tags have VR? So this function should be able to get it based on it right. Also do you think it should return null versus throwing exception?

Reply:
DicomElement(https://github.com/fo-dicom/fo-dicom/blob/development/DICOM/DicomElement.cs) has VR specified, while DicomTag is not.
Based on definition of DicomTag(https://github.com/fo-dicom/fo-dicom/blob/development/DICOM/DicomTag.cs), it has a bunch of candidate VRs -- defined in DictionaryEntry.